.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,53 @@
+name: test
```

Comment:
Why is file effectively renamed to `pythonapp.yml`? Do we expect do add code in golang here?